drivers/nuopc/ocn_comp_nuopc.F90

```diff
@@ -220,13 +219,13 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
     call ESMF_GridCompGet(gcomp, vm=vm, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-    call ESMF_VMGet(vm, mpiCommunicator=lmpicom, localPet=iam, rc=rc)
+    call ESMF_VMGet(vm, mpiCommunicator=lmpicom, localPet=my_task, rc=rc)
```

Comment:
This change looks suspicious to me. It looks like it is using the ESMF API to set `my_task`, which is a module variable in `communicate.F90` and is set there (via a setting of `POP_myTask` in `POP_CommMod.F90`). The previous code didn't modify POP's `my_task` variable.